code/src/index.css

```diff
@@ -11,3 +11,19 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+body {
+  background-color: #1f4e5f;
+}
+
+.done {
+  color: rgba(121, 168, 169, 0.4);
+}
```

Comment:
I think your conditional styling looks good, but if you want to convert everything to styled components, you can style classes like this:
```
export const SpanTextBold = styled.span`
  font-size: 14px;
  font-weight: bold;
  &.done {
  color: rgba(121, 168, 169, 0.4);
}
`
export const SpanTextItem = styled.span`
  font-size: 14px;
  &.done {
  color: rgba(121, 168, 169, 0.4);
}
`
```

or even cleaner - pass the done state along to the styled component:
`<SpanTextBold done={item.done}>`
and then style it differently according to done state:
```
export const SpanTextBold = styled.span`
  font-size: 14px;
  font-weight: bold;
  color: ${props => props.done ? "rgba(121, 168, 169, 0.4)" : "color for not done state"};
`
```